pkg/app/piped/planpreview/kubernetesdiff.go

```diff
@@ -79,9 +79,10 @@ func (b *builder) kubernetesDiff(
 		}
 	}
 
-	result, err := provider.DiffList(oldManifests, newManifests,
+	result, err := provider.DiffList(
+		oldManifests,
+		newManifests,
 		diff.WithEquateEmpty(),
-		diff.WithIgnoreAddingMapKeys(),
```

Comment:
We don't need this option because plan-preview is comparing between 2 commits, not using the live state.